.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: js-util CI
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - packages/**
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install dependencies
+      working-directory: packages/db-model
```

Comment:
Hardcoding the path here because there is not a straight forward way to detect changes in monorepos.

Reply:
:+1: